test/rest_api_test.js

```diff
@@ -68,35 +62,101 @@ describe('REST API request', function() {
         assert(res.body[0].age);
         assert(res.body[0].id);
         done();
-      });    
+      });
   });
 
-
-  it('should return review by customer 1', function(done) {
-    json('get', '/api/customers/1/reviews')
+  it('should return orders by customer 4', function(done) {
+    json('get', '/api/customers/4/orders')
       .expect(200, function(err, res){
         assert(Array.isArray(res.body));
         assert.notEqual(res.body.length, 0);
-        assert(res.body[0].product);
-        assert(res.body[0].star);
+        assert(res.body[0].description);
+        assert(res.body[0].date);
         assert(res.body[0].id);
-        assert(res.body[0].authorId);
+        assert(res.body[0].customerId);
         done();
-      });     
+      });
   });
 
-  it('should return review by customer 1', function(done) {
-    json('get', '/api/customers/1/orders')
-      .expect(200, function(err, res){
+  it('should return only 2 customers', function(done) {
+    json('get',
```

Comment:
why is this on two lines, put the url back on the same line to be consistent with the code above.